distbench_busybox.cc

```diff
@@ -259,9 +260,25 @@ int MainTestSequencer(std::vector<char*>& arguments) {
 }
 
 int MainNodeManager(std::vector<char*>& arguments) {
-  if (!AreRemainingArgumentsOK(arguments, 0, 0)) return 1;
+  int preassigned_node_id = -1;
+  if (!arguments.empty()) {
+    if (!strncmp(arguments[0], "node", 4)) {
+      preassigned_node_id = atoi(arguments[0] + 4);
+      if (absl::StrCat("node", preassigned_node_id) != arguments[0] ||
+          preassigned_node_id < 0) {
+        std::cerr << "node_manager expected nodeN as pre-assigned_node_id\n";
```

Comment:
Is this better than writing to logs?

Reply:
This is the CLI handling code people expect the output on stdout.

Reply:
actually stderr :)